src/gui/advancedsettings.cpp

```diff
@@ -418,7 +421,10 @@ void AdvancedSettings::loadAdvancedSettings()
     // Download tracker's favicon
     cb_tracker_favicon.setChecked(mainWindow->isDownloadTrackerFavicon());
     addRow(DOWNLOAD_TRACKER_FAVICON, tr("Download tracker's favicon"), &cb_tracker_favicon);
-
+    // Save path history length
+    spinDonwloadPathHistoryLength.setRange(0, INT_MAX);
```

Comment:
INT_MAX???

Reply:
What would be enough for everyone? I don't know... I understand that INT_MAX is certainly to high limit. INT_MAX/2?

Reply:
Changed to 100.

Reply:
I think 100 is more than enough. If too many of them, it is easier to choose the path using a dialog, instead of looking for in "this dump".